lib/customizerForMergeWith.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'underscore';
```

Comment:
Uh, we're using underscore to customize lodash? That's weird. I'd suggest just doing a vanilla JS check for `isArray()`. It looks like it's as simple as:

```
Array.isArray(arr);
```